starwars/src/App.js

```diff
@@ -1,7 +1,31 @@
-import React from 'react';
+import React, { useState, useEffect }from 'react';
 import './App.css';
+import axios from 'axios'
+import Card from './components/profile';
+import styled from 'styled-components';
+
+const styledDiv = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+`
 
 const App = () => {
+  const [results, setResults] = useState([]);
+
+  useEffect(() => {
+    function getData(){
+      axios
+        .get(`https://swapi.co/api/people/`)
+        .then(res => {
+          setResults(res.data.results);
+        })
+        .catch(err => console.log(err))
+    }
+    getData();
```

Comment:
the pattern I'm seeing is that you're writing a function inside the useEffect cB and then invoking it.  Why that pattern over just writing the axios request inside the useEffect cB?